lib/raven/integrations/rails/middleware/debug_exceptions_catcher.rb

```diff
@@ -6,8 +6,10 @@ def self.included(base)
           base.send(:alias_method_chain, :render_exception, :raven)
         end
 
-        def render_exception_with_raven(env, exception)
+        def render_exception_with_raven(env_or_request, exception)
+          env = env_or_request.respond_to?(:env) ? env_or_request.env : env_or_request
           Raven::Rack.capture_exception(exception, env)
+        ensure
           render_exception_without_raven(env, exception)
```

Comment:
shouldn't you be passing the original argument `env_or_request` instead of `env`?